domains/authorization/Proxy/Controllers/CertificatesController.cs

```diff
@@ -32,6 +32,25 @@ public async Task GetCertificatesCursor([FromQuery] GetCertificatesQueryParamete
         await ProxyClientCredentialsRequest("v1/certificates/cursor", organizationId);
     }
 
+    /// <summary>
+    /// Gets a single certificate
+    /// </summary>
+    /// <response code="200">Returns a certificate.</response>
+    /// <response code="401">If the user is not authenticated.</response>
```

Comment:
Ah I think you need to add it here in the summary

Reply:
fixed